lib/src/package_utils.dart

```diff
@@ -33,3 +33,50 @@ String getPackageDescription(String directoryName) {
 
 String getPackageVersion(String directoryName) =>
     _getPubspec(directoryName)['version'];
+
+const String SDK_INTRO = """
```

Comment:
@keertip @sethladd https://codereview.chromium.org/1015013006/
Once this is merged, I'll do a check to see if the SDK contains one, and if so, we will use it, otherwise, we wont.